templates/openstackconfiggenerator/config/16.2/rendered-tripleo-config.yaml

```diff
@@ -123,4 +123,38 @@ parameter_defaults:
 {{- end }}
 {{- end }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+  #
+  # CtlplaneNetworkAttributes
+  CtlplaneNetworkAttributes:
+    {{- range $netname, $net := .NetworksMap }}
+    {{- $netname_lower := lower $net.Name }}
```

Comment:
netname_lower is not used and should be not required as there is $net.NameLower? Think we can also remove it from the 17.0 templates

Reply:
done